dspace/config/crosswalks/DIM2DataCite.xsl

```diff
@@ -89,11 +89,13 @@
                     <xsl:when test="//dspace:field[@mdschema='dc' and @element='contributor' and @qualifier='author']">
                         <xsl:apply-templates select="//dspace:field[@mdschema='dc' and @element='contributor' and @qualifier='author']" />
                     </xsl:when>
-                    <xsl:otherwise>
-                        <creator>
-                            <creatorName>(:unkn) unknown</creatorName>
-                        </creator>
-                    </xsl:otherwise>
+<!-- If your repository does not have any creator/author metadata included, but you still wish to register items in
```

Comment:
I don't like this change, sorry. I explained my thoughts in a comment in the bug issue: https://github.com/DSpace/DSpace/issues/9640#issuecomment-3496498740